auth-service/src/main/java/com/petclinic/auth/User/UserController.java

```diff
@@ -65,7 +67,9 @@ public User getUser(@PathVariable long userId) {
     }
 
     @PostMapping
-    public UserPasswordLessDTO createUser(@RequestBody @Valid UserIDLessRoleLessDTO dto) {
+    public UserPasswordLessDTO createUser(
+            @RequestBody @Valid UserIDLessRoleLessDTO dto,
+            BindingResult bindingResult) {
```

Comment:
Where is this bindingResult used?

Reply:
It is not nor should it be.
For some ungodly reason, Java X Validation does NOT throw the appropriate exception if you do not take in the `BindingResult` in your method params.